flow360/component/project.py

```diff
@@ -0,0 +1,563 @@
+        elif root_type == RootType.VOLUME_MESH:
+            draft = VolumeMeshV2.from_file(file, name, solver_version, length_unit, tags)
+            root_asset = draft.submit()
+        if not root_asset:
+            raise Flow360ValueError(f"Couldn't initialize asset from {file}")
+        project_id = root_asset.project_id
+        project_api = RestApi(ProjectInterface.endpoint, id=project_id)
+        info = project_api.get()
+        if not info:
+            raise Flow360ValueError(f"Couldn't retrieve project info for {project_id}")
+        project = Project(metadata=ProjectMeta(**info), solver_version=root_asset.solver_version)
+        if root_type == RootType.GEOMETRY:
+            project._geometry = root_asset
+        elif root_type == RootType.VOLUME_MESH:
+            project._volume_mesh = root_asset
+        return project
+
+    @classmethod
+    def from_cloud(cls, project_id: str):
+        """
+        Loads a project from the cloud.
+
+        Parameters
+        ----------
+        project_id : str
+            ID of the project.
+
+        Returns
+        -------
+        Project
+            An instance of the project.
+
+        Raises
+        ------
+        Flow360WebError
+            If the project cannot be loaded from the cloud.
+        Flow360ValueError
+            If the root asset cannot be retrieved for the project.
+        """
+        project_api = RestApi(ProjectInterface.endpoint, id=project_id)
+        info = project_api.get()
+        if not isinstance(info, dict):
+            raise Flow360WebError(f"Cannot load project {project_id}, missing project data.")
+        if not info:
+            raise Flow360WebError(f"Couldn't retrieve project info for {project_id}")
+        meta = ProjectMeta(**info)
+        root_asset = None
+        root_type = meta.root_item_type
+        if root_type == RootType.GEOMETRY:
+            root_asset = Geometry.from_cloud(meta.root_item_id)
+        elif root_type == RootType.VOLUME_MESH:
+            root_asset = VolumeMeshV2.from_cloud(meta.root_item_id)
+        project = Project(metadata=meta, solver_version=root_asset.solver_version)
+        if not root_asset:
+            raise Flow360ValueError(f"Couldn't retrieve root asset for {project_id}")
+        if root_type == RootType.GEOMETRY:
+            project._geometry = root_asset
+        elif root_type == RootType.VOLUME_MESH:
+            project._volume_mesh = root_asset
+        return project
+
+    def _check_initialized(self):
+        """
+        Checks if the project instance has been initialized correctly.
+
+        Raises
+        ------
+        Flow360ValueError
+            If the project is not initialized.
+        """
+        if not self.metadata or not self.solver_version:
+            raise Flow360ValueError(
+                "Project is not initialized - use Project.from_file or Project.from_cloud"
+            )
+
+    def get_simulation_json(self) -> dict:
```

Comment:
```suggestion
    def get_root_simulation_json(self) -> dict:
```

Reply:
Fixed in [9ba5587](https://github.com/flexcompute/Flow360/pull/511/commits/9ba5587bee8a37191788f1094fe8503dd5cda41d)